PokerChallenge/PokerChallenge(2)/PokerHandTypes/OnePair.cs

```diff
@@ -0,0 +1,74 @@
+﻿using PokerChallenge2.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PokerChallenge2
+{
+    public static class OnePair
+    {
+        public static bool IsOnePair(this PokerHand hand)
+        {
+            var result = false;
+
+            foreach(var card in hand.Cards)
+            {
+                if(hand.Cards.Where(c => c.Value == card.Value).Count() == 2)
+                {
+                    result = true;
+                }
+            }
+
+            return result;
+        }
```

Comment:
This could be entirely simplified into a single LINQ statement as well. You are half way there. Look at my previous comments for more information.